content/uk/docs/reference/glossary/pod.md

```diff
@@ -0,0 +1,23 @@
+---
+# title: Pod
+title: Под
```

Comment:
nit: I'd call this “Pod” because it's an API object. (That's also what https://github.com/kubernetes/website/blob/master/content/ru/docs/reference/glossary/pod.md does)

Reply:
We have a localization rule for this situation
https://github.com/kubernetes/website/pull/20020/files#diff-11dd086c148eeba321233dba5426004eR25
https://github.com/kubernetes/website/pull/20020/files#diff-11dd086c148eeba321233dba5426004eR94

English variant is not so good because in Ukrainian we have changeable end of the word, so we need use next constructions:
```
Пода   | Pod'а
Поди   | Pod'и
Подами | Pod'ами
Подів  | Pod'ів
...
```
With English "root of the word" it looks not cool.

Also, in local chats, folks very often use transliterated words for popular constructions so this not be something new for them.
In other hand, we can't transliterate all thing, like `Deployment -> Деплоймент` because it has established analogue in Ukrainian `Deployment -> Розгортання` that will be not understandable across the world

Reply:
@MaxymVlasov my only is that word "Под" doesn't exist in Ukrainian. IMO, I'd prefer using the "Pod'_ending_" constructions.

The same matters for all the well-established namings of the Kubernetes objects, including Deployment, Service, etc.

cc @anastyakulyk

Reply:
@idvoretskyi thanks, we have revised the texts according to your suggestions